### ISSUE TITLE
Node 8 support dropped in minor version?

### ISSUE THREAD

**Issue by IanVS** [applitools/eyes.sdk.javascript1#230]
I am stuck on node 8 for the time being, and attempted to upgrade to `@applitools/eyes-storybook@3.8.5` from `3.2.5`, but I am getting an error:

```
error @applitools/feature-flags@2.1.1: The engine "node" is incompatible with this module. Expected version ">=10.3.0". Got "8.17.0"
```

I believe that normally dropping support for a node version is considered a breaking change, and so the major version should have been bumped to `4.x`, right?  

In the meantime, can you tell me what the last version to support Node 8 is? Thanks!

**Comment by amitzur** [comment]
@IanVS we should be supporting Node.js version 8, so this is considered a bug. It will be resolved shortly.

**Comment by amitzur** [comment]
@IanVS issue is resolved in version `3.8.6`
